### ISSUE TITLE
[testnet] withdrawal is not called automatically.

### ISSUE THREAD

**Issue by conr2d** [Game-X-Coin/gxc.contracts#71]
Moved from Game-X-Coin/gxc#33 by @realcom 

Test team call many withdraw request but 9/10 withdrawal request(delayed transaction) is not called.

You can find this issues by checking testuserxxxx(1~5) of Q@catfight, W@catfight, E@catfight, R@catfight.

some examples are

testuser1341 (R@catfight, E@catfight)
testuser3251 (R@catfight, W@catfight)
testuser4135 (E@catfight, R@catfight)
testuser3415 (R@catfight, W@catfight)

**Comment by realcom** [comment]
Test needed?

**Comment by conr2d** [comment]
I don't think so, but please close this issue after checking whether your problem is solved completely.

**Comment by realcom** [comment]
I have tried same test and pick 20 random cases. The percentage of not calling withdrawal has decreased but it occasionally existed.

Some cases are

https://monitor.gxc.co.in/accounts/testuser3422 (E, R)
https://monitor.gxc.co.in/accounts/testuser1255 (E)
https://monitor.gxc.co.in/accounts/testuser2131 (R)
https://monitor.gxc.co.in/accounts/testuser3434 (R)
